mashup.css

```diff
@@ -0,0 +1,88 @@
+body {
+	font-family: Frutiger,"Frutiger Linotype",Univers,Calibri,"Liberation Sans","Nimbus Sans L",Tahoma,Geneva,"Helvetica Neue",Helvetica,Arial,sans-serif;
+	color: #000066;
+}
+p {
+	color: #0000CC;
+}
+header {
+	padding-bottom: 20px;
+}
+header h1 {
+	font-family: 'Lobster', cursive;
+	font-size: 50px;
+	text-align: center;
+}
+header p {
+	width: 90%;
+	margin: 0 auto;
+	font-size: 18px;
+	text-align: center;
+}
+div.category {
+	width: 90%;
+	margin: 0 auto;
+	font-size: 30px;
+	border: 1px solid #000066;
+	background-color: #ffffb3;
+}
+div.category span.open {
+	padding-left: 10px;
+	cursor: pointer;
+}
+div.category span.open:hover {
+	letter-spacing: 10px;
+}
+div.category span.close {
+	font-family: 'PT Sans', sans-serif;
+	padding-right: 10px;
+	visibility: hidden;
+	float: right;
+	cursor: pointer;
+}
+section[id*="news"] {
```

Comment:
Not to mention being more conventional, when there isn't an advantage to going against it.